core/src/main/java/discord4j/core/object/data/stored/MessageBean.java

```diff
@@ -176,6 +176,7 @@ public void setAuthor(@Nullable final UserBean author) {
         this.author = author;
     }
 
+    @Nullable
```

Comment:
Why is this nullable ?

Reply:
https://discordapp.com/developers/docs/resources/channel#message-object

content is non-null

Reply:
But in Message.java the content in a Optional.ofNullable (with a note) for this i add this Nullable (only for avoid the warn of IDE)

Reply:
This is an artifact during 3.0 development that stuck to avoid breaking API in a patch version.

If I remember correctly, for some reason at the time content could be `null`.

So, can you remove the annotation?

Reply:
Removed